sys/net/gnrc/netif/gnrc_netif_device_type.c

```diff
@@ -143,7 +145,7 @@ void gnrc_netif_ipv6_init_mtu(gnrc_netif_t *netif)
                                    &tmp, sizeof(tmp));
             assert(res == sizeof(tmp));
 #ifdef MODULE_GNRC_SIXLOWPAN
-            netif->ipv6.mtu = IPV6_MIN_MTU;
+            netif->ipv6.mtu = MAX(IPV6_MIN_MTU, tmp);
```

Comment:
This is not a good idea, as this will lead to fragmentation in the wider internet. Better cap it at 1500.

Reply:
Or better: we should implement Path MTU discovery ;-P

Reply:
I'll see what I can do for Path MTU discovery, but that should be unrelated.
Btw.: SLIP already does `MTU:65535  HL:64  RTR`

Reply:
Yeah. My bad. This comment was more loud thinking and inner alarm bells ringing ;-).